src/main/java/com/flightstats/hub/util/SecretsFilter.java

```diff
@@ -0,0 +1,15 @@
+package com.flightstats.hub.util;
+
+
+import java.util.Arrays;
+import java.util.List;
+
+public class SecretsFilter {
+    private final static String redacted = "[REDACTED]";
+    private final static List<String> matchers = Arrays.asList("app_key", "api_key", "password");
+
+    public String redactionFilter(String key, String property) {
```

Comment:
redact would be a better name since the class name already has filter.

Reply:
👍